src/main/java/com/autotune/analyzer/services/GenerateRecommendations.java

```diff
@@ -0,0 +1,188 @@
+/*******************************************************************************
```

Comment:
This needs to be added to the servlets in [services](https://github.com/kruize/autotune/blob/master/src/main/java/com/autotune/analyzer/Analyzer.java).

Reply:
Added now